sql/src/main/java/io/crate/action/sql/SQLOperations.java

```diff
@@ -107,26 +119,28 @@ public Session createSession(@Nullable String defaultSchema) {
 
         private final Executor executor;
         private final String defaultSchema;
-        private final UUID jobId;
 
+        private UUID jobId;
         private List<DataType> paramTypes;
         private Statement statement;
         private Analysis analysis;
-        private Plan plan;
-        private ResultReceiver rowReceiver;
         private List<DataType> outputTypes;
         private String query;
 
         private Throwable throwable = null;
+        private List<List<Object>> bulkParams = new ArrayList<>();
```

Comment:
Maybe rename bulkParams to params since are used for the simple operation as well.